modules/ROOT/pages/java-support.adoc

```diff
@@ -0,0 +1,134 @@
+= Java Support
+
+MuleSoft is adopting Java’s long-term-support (LTS) release cadence, beginning with Mule Runtime 4.6, which adds support for Java 17 LTS. 
+
+Many third-party vendors are discontinuing support for Java 8, which impacts the MuleSoft product stack. Adopting this new release cadence provides greater security, compliance, and performance.
+
+This change has no immediate impact to Mule applications and Mule Gateway policies and proxies that are running on currently supported Mule versions, or on applications that are upgraded to Mule Runtime 4.6, as existing Java 8 apps will continue to run on Java 8 until February 2026, giving you time to plan for your upgrade to Java 17.
+
+Java 17 upgrade planning includes evaluating connectors, integration apps, Mule Gateway policies, and proxies, as all of these components rely on Java. 
+
+== Mule Runtime Java Support
+
+The following table shows the timeline for Java 8 and Java 17 support:
```

Comment:
i know it is in the note, but should we call out Java 11 support in the table too?

Reply:
I don't know yet. I brought this up before and everyone was resistant.

Reply:
Mule runtime does [support Java 11](https://docs.mulesoft.com/mule-runtime/latest/hardware-and-software-requirements#required-software). So in this table, whenever JDK 8 appears, we might want to also add JDK 11 as they have the same support period.